0.16/files/scenario.sh

```diff
@@ -11,7 +11,7 @@ id
 mkdir -p "$SAVES"
 mkdir -p "$CONFIG"
 mkdir -p "$MODS"
-mkdir -p "$SCENARIO"
+mkdir -p "$SCENARIOS"
```

Comment:
`SERVER_SCENARIO`?

Reply:
The `$SERVER_SCENATIO` directory already exists - why would we need to make it?
Alternatively `$SCENATIOS` is defined by the container, unsure if this needs to exist in order to launch the server

Reply:
I'm just asking, because you renamed `$SCENARIO` into `$SCENARIOS` and above/below into `$SERVER_SCENARIO`.
Which confuses me.

Reply:
I would just rename all variables to $SCENARIOS if therei s no conflict.

Reply:
That would conflict

$SCENARIOS is the folder for storing scenarios /appdata/factorio/scenarios defined by the docker container just as "$SAVES" or "$CONFIG"
$SERVER_SCENARIOS is a specific scenario /user/brandon/scenario_1.scn passed in by the command line

If you look at the docker entry-point, the directories are all made before running the container:
mkdir -p "$SAVES"
mkdir -p "$CONFIG"
mkdir -p "$MODS"
mkdir -p "$SCENARIOS"

Thus I am assuming that for the scenario.sh entry-point, these directories will need to be made as well

Reply:
Ok this is clear now :+1: